app/services.py

```diff
@@ -52,3 +55,82 @@ def delete_user(db: Session, profile_id: str):
     db.commit()
     return f"User {profile_id} has been deleted"
 
+
+def save_awards_data(db: Session, f: UploadFile = File(...)):
+    f = os.path.realpath(os.path.join("fixtures", f.filename))
+    awards_file = openpyxl.load_workbook(f)
+    awards_sheet = awards_file.active
+    rows = awards_sheet.iter_rows(13, 3000, 0, 43, values_only=True)
```

Comment:
so here you've fixed the row counts. But each month users are going to upload a new file. There may be more or less rows of data in the file than the last time. So this needs to updated to work such that you iterate through the rows, and check for some condition that tells you to stop iterating.